src/main/java/org/littleshoot/proxy/Launcher.java

```diff
@@ -134,10 +133,10 @@ private static void printHelp(final Options options,
     }
 
     private static void pollLog4JConfigurationFileIfAvailable() {
-        File log4jConfigurationFile = new File("src/test/resources/log4j.xml");
-        if (log4jConfigurationFile.exists()) {
-            DOMConfigurator.configureAndWatch(
-                    log4jConfigurationFile.getAbsolutePath(), 15);
-        }
+//        File log4jConfigurationFile = new File("src/test/resources/log4j.xml");

```

Comment:
is this important?

Reply:
Only if it's run standalone. We prob can drop this file at some point.